var/spack/repos/builtin/packages/py-python-dateutil/package.py

```diff
@@ -20,7 +21,7 @@ class PyPythonDateutil(PythonPackage):
     version('2.4.0', sha256='439df33ce47ef1478a4f4765f3390eab0ed3ec4ae10be32f2930000c8d19f417')
     version('2.2',   sha256='eec865307ebe7f329a6a9945c15453265a449cdaaf3710340828a1934d53e468')
 
-    depends_on('python@2.7:2.8,3.4:', type=('build', 'run'))
+    depends_on('python@2.7:2.8,3.3:', when='@2.7.5:2.8.2', type=('build', 'run'))
```

Comment:
```suggestion
    depends_on('python@2.7:2.8,3.3:', when='@2.7.5:', type=('build', 'run'))
```
Presumably 2.8.3 will also support the same versions